tests/ceph_ansible/purge_dashboard.py

```diff
@@ -0,0 +1,56 @@
+""" Module to purge ceph dashboard"""
+import logging
+import re
+
+log = logging.getLogger(__name__)
+
+
+def run(**kw):
+    """
+
+    Args:
+       param kw:
+       - ceph_nodes: ceph node list representing a cluster.
+       - config: (optional)
+         - ansible-dir: path to ansible working directory, default is /usr/share/ceph-ansible.
+         - inventory: ansible inventory file, default is hosts.
+         - playbook-command: ansible playbook command string.
+
+
+    return: 0 on sucess, non-zero for failures
+    """
+    log.info("Running purge-dashboard test")
+    ceph_nodes = kw.get("ceph_nodes")
+    installer_node = None
+    ansible_dir = "/usr/share/ceph-ansible"
```

Comment:
As mentioned in the doc string `ansible-dir`, value is not considered here.
I think it would be 
```
ansible_dir = kw.get("ansible-dir", "/usr/share/ceph-ansible")
```

Reply:
Since these path is very static, So i thought it's better to hard code in module itself,
Instead of adding everytime in needed test suites 
@sunilkumarn417 Are you ok with this?

Reply:
Oh okay, review comment was based on `ansible-dir` key-value in `kw` doc string.
If it is not changing anytime, can we take out that parameter from doc string.